### PR TITLE
Add pre- and post-event forms to repo

### PR DESCRIPTION
**Pull request by isabela-pf**
- [x] Check if there is any specific information needed in the attendee forms to be passed on to get people their invites
- [x] Write speaker event attendee forms
- [x] Write speaker event feedback forms
- [x] Write sprint event attendee forms
- [x] Write sprint event feedback forms

The non-checked options are currently empty links. 

Please note
> For now, these forms are not taking public responses to make sure we don't get information before we are ready. They all need to be made public pre-event.

### CODE REVIEW

**Comment by isabela-pf** [comment]
@tonyfast @trallard here's the [draft form of the speaker event sign up](https://docs.google.com/forms/d/e/1FAIpQLSf0IgPYgf0di8eM6HP1qbqduPTeGEFadXZRSIDYEXlEOgA_bw/viewform?usp=sf_link). This is open for feedback, especially if we need some information specifically for Eulerr.

**Comment by isabela-pf** [comment]
I've just updated all the pre- and post-event forms linked in this PR! The main changes were to account for new event dates, but I also added or clarified some questions. 

I'd love to get another person reviewing this, though. I'm sure there's something I've missed.

**Comment by trallard** [comment]
Feedback:

- [In the sprint sign-up](https://docs.google.com/forms/d/e/1FAIpQLSfyB3NAbNcYLIQX8ODI5m86R4hnt0T5z2HP-rNLMUmbRciyZg/viewform) you have the following question:

> Do you have dependents that you may need to care for during the event?

This is an important question but it also feels sketchy without background. We should add descriptive text along the lines "We aim to make this an inclusive event for all members of our community to take part. For this purpose, we have set aside limited funding to support those with care responsibilities, each case will be evaluated on a case-by-case basis by the organising committee based on the information provided in this section."

**Comment by trallard** [comment]
In the post-event feedback maybe reword

>Do you have any other feedback for the planning team?
 What could the planning team do better for future events?

Folks are usually better at giving feedback when asking for well-scoped feedback

**Comment by isabela-pf** [comment]
> This is an important question but it also feels sketchy without background

I thought the same thing. Thanks for giving feedback on what was an appropriate way to frame it.

I've made the above changes (and some minor copy edits found from reviewing with fresh eyes). I can't thank you enough for the review!

I don't think anyone else is going to review this (and it is still editable once we share the link), so I will 

- [ ] update the linked forms to have public access
- [ ] add the new links to the other open PRs #4 and #8 

and then merge this. Thanks again to everyone who reviewed this on GitHub and elsewhere!

**Review by trallard — APPROVED** [review]
🚀  Let's merge this
